src/PcapTools.jl

```diff
@@ -4,14 +4,15 @@ using Dates
 using Mmap
 using UnixTimes
 using UnsafeArrays
+using CRC32
 
 export PcapHeader, RecordHeader
 export PcapRecord
 export PcapReader, PcapStreamReader, PcapBufferReader
 export PcapWriter, PcapStreamWriter
 export LINKTYPE_NULL, LINKTYPE_ETHERNET
 export splitcap
-export pcap_has_fcs, ETHERNET_FCS_LENGTH
+export pcap_has_fcs, fcs, compute_fcs, ETHERNET_FCS_LENGTH
```

Comment:
`fcs` is no longer an exported function
```suggestion
export pcap_has_fcs, compute_fcs, ETHERNET_FCS_LENGTH
```